public/landing/landing.styl

```diff
@@ -113,9 +127,54 @@ body
 		line-height main-app-name-font-size / 2
 		box-shadow 0 1px 0 0 darken(green-main, 50%)
 	
+	.app-description
+		display flex
+		flex-flow row nowrap
+		justify-content space-between
+		align-items stretch
+		width main-space-width
+		margin-left main-space-left-offset
```

Comment:
It's not recommended to use `flexbox` for everything because of performance issues. Especially it's visible in non-webkit browsers and mobile ones.